apps/dashboard/app/(app)/logs-v2/components/controls/components/logs-datetime/components/suggestions.tsx

```
@@ -0,0 +1,102 @@
+        onChange(options[focusedIndex].id);
+        break;
+    }
+  };
+
+  return (
+    <div
+      className={cn("flex flex-col justify-center gap-1.5", className)}
+      onKeyDown={handleKeyDown}
+      role="listbox"
+      aria-label="Time range options"
+    >
```

Biome (1.9.4)

[error] The HTML element with the interactive role "listbox" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "listbox".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)